example/python/ed25519.py

```diff
@@ -43,6 +43,9 @@
 import operator
 import sys
 
+if sys.version_info < (3, 6):
```

Comment:
Ehm, what about `>= (3, 6)`, ain't it require an import as well?

Reply:
You are right. Since 3.6 no additional imports are required.